test/src/module/command/archiveCommonTest.c

```diff
@@ -33,15 +33,15 @@ testRun(void)
         // -------------------------------------------------------------------------------------------------------------------------
         String *segment = strNew("000000010000000100000001");
 
-        TEST_RESULT_BOOL(archiveAsyncStatus(archiveModePush, segment, false), false, "directory and status file not present");
-        TEST_RESULT_BOOL(archiveAsyncStatus(archiveModeGet, segment, false), false, "directory and status file not present");
+        TEST_RESULT_BOOL(archiveAsyncStatus(archiveModePush, segment, false, true), false, "directory and status file not present");
+        TEST_RESULT_BOOL(archiveAsyncStatus(archiveModeGet, segment, false, true), false, "directory and status file not present");
```

Comment:
At some point, I would rather this function was entirely tested by the archive-get/archive-push modules since they are better tests end to end, but since we still have the separate test it makes sense to keep it up to date. Test added in 686b0088.